### PR TITLE
New weltsparen.de logo

### PR DESCRIPTION
**Pull request by BaumiCoder**
1. I got svg from website with logo and writing. Link to the file: https://cdn.weltsparen.de/app/themes/sg-core-theme/assets/lib/img/Logo_Raisin.svg
2. I cuted the logo out with Inkscape.
3. I optimized it with SVGOMG (default settings).

issue: #7561

Here at github the new logo looks smaller. I hope that isn't a problem. (As far as I know svg files have not a real size in pixel, aren't?)

### CODE REVIEW

File: img/w/weltsparen.de.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...37a2.4 2.4 0 0 1-4.78 0c0-1.3 1.08-2.37 2.38-2.37" class="st0"/></g><path id="rect18" d="M74.95 6.73h3.99v14.05h-3.99z" class="st0"/><g id="g22"><path id="path20" d="M82.1 6.73h4v1.82a5.2 5.2 0 0 1 4.38-2.27c1.24 0 3.07.35 4.18 1.72.98 1.2 1.11 2.6 1.11 3.95v8.83h-3.99v-7.66c0-.78-.03-2-.75-2.75a2.56 2.56 0 0 0-1.86-.72c-1.05 0-1.86.36-2.45 1.23a4.8 4.8 0 0 0-.62 2.73v7.17H82.1V6.73Z" class="st0"/></g><path id="polygon24" d="M31.11 14.03v6.75h-3.99V6.73h3.99z" class="st0"/><g id="g28"><path id="path26" d="M33.83 6.27a2.4 2.4 0 0 1 2.4 2.37 2.4 2.4 0 0 1-4.78 0 2.4 2.4 0 0 1 2.38-2.37" class="st0"/></g><linearGradient id="SVGID_1_" x1="3.32" x2="18.06" y1="18.02" y2="3.28" gradientUnits="userSpaceOnUse"><stop id="stop30" offset="0" style="stop-color:#156cc4"/><stop id="stop32" offset=".16" style="stop-color:#1368bb"/><stop id="stop34" offset=".41" style="stop-color:#0f5ca3"/><stop id="stop36" offset=".73" style="stop-color:#07487b"/><stop id="stop38" offset="1" style="stop-color:#003554"/></linearGradient><path id="path41" d="M10.62.2a10.52 10.52 0 0 0-8.7 16.4l5.88-5.88H2.9v-4H14.63v11.71h-4v-4.9L8.17 16l-3.44 3.44A10.52 10.52 0 1 0 10.63.2z" style="fill:url(#SVGID_1_)"/></g></svg>
```
Comment (hkamran80) [inline img/w/weltsparen.de.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 21.01 21.04"><defs><linearGradient id="a" x1="3.32" x2="18.06" y1="18.02" y2="3.28" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#156cc4"/><stop offset=".16" stop-color="#1368bb"/><stop offset=".41" stop-color="#0f5ca3"/><stop offset=".73" stop-color="#07487b"/><stop offset="1" stop-color="#003554"/></linearGradient></defs><path fill="url(#a)" d="M10.5 0a10.52 10.52 0 0 0-8.7 16.4l5.88-5.88h-4.9v-4H14.5v11.71h-4v-4.9L8.05 15.8 4.6 19.24A10.52 10.52 0 1 0 10.5 0Z"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to server and browser caches.

**Comment by BaumiCoder** [comment]
@hkamran80 Thank you for your help. I saw, that your suggestion removes the size in pixel of the svg image. 
Only out of own interest: How did you do that? Which tool to you use to make this changes?

**Comment by hkamran80** [comment]
@BaumiCoder You're welcome! I ran the logo through [SVGOMG](https://jakearchibald.github.io/svgomg/), with all the settings active, except for "Remove xmlns". Then I lowered the two precision sliders until they were as low as possible without sacrificing too much quality.

**Pull request merged**